zypper.spec.cmake

```diff
@@ -175,8 +176,9 @@ rm -rf "$RPM_BUILD_ROOT"
 %license COPYING
 %endif
 %config(noreplace) %{_sysconfdir}/zypp/zypper.conf
-%config(noreplace) %{_sysconfdir}/logrotate.d/zypper.lr
-%config(noreplace) %{_sysconfdir}/logrotate.d/zypp-refresh.lr
+%dir %{_distconfdir}/logrotate.d
+%{_distconfdir}/logrotate.d/zypper.lr
+%{_distconfdir}/logrotate.d/zypp-refresh.lr
```

Comment:
If `%{_distconfdir}` is unset or `== %{_sysconfdir}`, the files must go `%config(noreplace) %{_sysconfdir}/...` as before.